src/test/java/org/folio/rest/impl/UsersAPITest.java

```diff
@@ -91,6 +93,15 @@ void postUsersException(VertxTestContext vtc) {
         })), null);
   }
 
+  @Test
+  void removeCustomField(VertxTestContext vtc) {
+    CustomFields customFields = new CustomFields().withAdditionalProperty("test", "");
+    User user = new User().withCustomFields(customFields);
+    new UsersAPI().removeCustomFieldIfEmpty(user);
+    vtc.verify(() -> assertTrue(user.getCustomFields().getAdditionalProperties().isEmpty()));
+    vtc.completeNow();
+  }
+
```

Comment:
It should be an API test. Please, check how API tests are written in other test cases.